custom_components/tuya_local/devices/aygrochy_inline_wifi_air_vent.yaml

```diff
@@ -0,0 +1,314 @@
+name: Aygrochy fan
+primary_entity:
+  entity: climate
+  dps:
+    - id: 2
+      name: preset_mode
+      type: string
+      mapping:
+        - dps_val: "ON"
+          value: "ON"
+        - dps_val: "OFF"
+          value: "OFF"
+        - dps_val: "TIMER"
+          value: "TIMER"
+        - dps_val: "AUTO"
+          value: "AUTO"
+        - dps_val: "ALARM"
+          value: "ALARM"
+    - id: 8
+      readonly: true
+      name: current_humidity
+      type: integer
+      range:
+        min: 0
+        max: 255
+      unit: "%"
+    - id: 9
+      readonly: true
+      name: current_temperature
+      type: integer
+      range:
+        min: -40
+        max: 255
+      unit: "F"
+    - id: 106
+      name: target_temp_high
+      type: integer
+      range:
+        min: 32
+        max: 194
+      unit: "F"
+    - id: 107
```

Comment:
Since the climate entity does not support high and low target humidity settings, probably this should be removed from here and the separate number entities below used.